### DISCUSSION TITLE
How do i exit mini player?

### DISCUSSION THREAD

**Discussion by nepalivai** [KRTirtho/spotube#1090] [Design]
As shown in the pic below there is no way to exit the mini player at least I can't see it. When I go to the mini player to go back I have to shut down the whole app and restart I will make it an issue but I want to know if there is some way or if it may be different in Mac/Linux.
<img width="439" alt="image" src="https://github.com/KRTirtho/spotube/assets/108126089/fc260e8f-157a-45ad-9651-7fa85947ab54">

**Comment by KRTirtho** [comment]
Click on the expanding arrow icon on the bottom right corner

**Reply by nepalivai** [reply]
ohh!!! that seems to be it. I think needs some modification as it can be hard to find it is the full screen icon after all.

**Reply by ottercynical** [reply]
There is no such arrow after hiding the UI on Windows 10 as of v3.9.0, Feb 24 2025.  There is no way to exit the miniplayer once the mini UI has been hidden.  There is no way to redisplay it.
